.github/workflows/push.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:11
+        ports:
+        - 5432:5432
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          host: localhost
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
```

Comment:
Think the rest of the stuff here is just extra. Doesn't hurt it have it, but if you wanted to you can just remove it and everything will still work.

E.g:

```
      postgres:
        image: postgres:11-alpine
        ports: ["5432:5432"]
        env:
          POSTGRES_PASSWORD: mysecretpassword
```
is all you really need